### PR TITLE
Update main.cpp

### PR DESCRIPTION
**Pull request by jsonwastaken**
For now I'm only using one function. The elements get filled with 1 or 2 but I'm also able to override the same element(Trying to fix this).
playField[] is the array you can check for a win.  There should be 8 win conditions for both sides.

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -1,7 +1,135 @@
...
+		cout << "Turn number: " << (i + 1) << endl;
+
+		if (i % 2 == 0)			//Checks who's turn it is	*(here only for testing)*
+			cout << "Player 1's turn." << endl;
+		else
+			cout << "Player 2's turn." << endl;
+
+		while (pause == true)
+		{
+
```
Comment (Kim1802) [inline main.cpp:54]: ask the user for what they need to input

File: main.cpp [hunk 2]
```diff
@@ -1,7 +1,135 @@
+/*
+
+Current Problem:
+
+How can I exclude the already filled elements so they cannot be overwritten, without using multiple (if else) statements...
+
```
Comment (Kim1802) [inline main.cpp:6]: have a second array bool check[9] (same size) to store if a space is used already, check this anytime the user puts the input

File: main.cpp [hunk 3]
```diff
@@ -1,7 +1,135 @@
...
 using namespace std;
 
-int main() {
+int input(string);  //Function used inside playField[] to pick which element to change
+
+//----------------------------------------Main----------------------------------------//
+int main()
+{
+
+	string r[9];		//Round counter
```
Comment (Kim1802) [inline main.cpp:24]: this is not counter to me from how you use it, more like a user input and should be named something descriptive

File: main.cpp [hunk 4]
```diff
@@ -1,7 +1,135 @@
...
+						//Element reference
+						//0,1,2,3,4,5,6,7,8,this last element holds error input, will always be zero.
+	int playField[10] = { 0,0,0,0,0,0,0,0,0,0 }; // 0 - emtpy,  1 - X,  2 - O
```
Comment (meomeo1802) [inline main.cpp]: why is there 10 not 9?

File: main.cpp [hunk 5]
```diff
@@ -1,7 +1,135 @@
...
+								
+					playField[input(r[i])] = 1; //Places 1 into array
+
+					if (input(r[i]) != 9)
+						pause = false;
+
+					else
+					{
+						playField[9] = 0;
+						cout << "Choose a correct box" << endl;
```
Comment (meomeo1802) [inline main.cpp]: prompt the user to input a correct value

**Pull request closed without merging**